apollo-runtime/src/main/java/com/apollographql/android/cache/normalized/sql/MySQLiteHelper.java

```diff
@@ -0,0 +1,42 @@
+package com.apollographql.android.cache.normalized.sql;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import android.util.Log;
+
+public class MySQLiteHelper extends SQLiteOpenHelper {
```

Comment:
Yeah this one I wasn't sure of. So if someone already is using sql they need to be either able to pass in their own Helper to the DAO or extend this one with own tables. Not sure which direction to go